functions/channelCapture/channelCaptureHandlers.private.ts

```diff
@@ -0,0 +1,531 @@
+    },
+  });
+
+  // const updated =
+  await updateChannelWithCapture(channel, {
+    userId,
+    botName,
+    botAlias,
+    controlTaskSid,
+    releaseType,
+    studioFlowSid,
+    releaseFlag,
+    memoryAttribute,
+    chatbotCallbackWebhookSid: chatbotCallbackWebhook.sid,
+  });
+
+  const handlerPath = Runtime.getFunctions()['helpers/lexClient'].path;
+  const lexClient = require(handlerPath) as LexClient;
+
+  const lexResponse = await lexClient.postText(context, {
+    botName,
+    botAlias,
+    userId,
+    inputText,
+  });
+
+  // Send message to trigger the recently created chatbot integration
+  await channel.messages().create({
+    body: lexResponse.message,
+    from: 'Bot',
+    xTwilioWebhookEnabled: 'true',
+  });
+};
+
+/**
+ * Send a message to the channel and add the chatbot after, so it will get triggered on the next response from the service user (used to trigger executions from system, like post surveys)
+ */
+const triggerWithNextMessage = async (
+  context: Context<EnvVars>,
+  channel: ChannelInstance,
+  {
+    userId,
+    botName,
+    botAlias,
+    inputText,
+    controlTaskSid,
+    releaseType,
+    studioFlowSid,
+    releaseFlag,
+    memoryAttribute,
+  }: CaptureChannelOptions,
+) => {
+  /** const messageResult = */
+  await channel.messages().create({
+    body: inputText,
+    xTwilioWebhookEnabled: 'true',
+  });
+
+  const chatbotCallbackWebhook = await channel.webhooks().create({
+    type: 'webhook',
+    configuration: {
+      filters: ['onMessageSent'],
+      method: 'POST',
+      url: `https://${context.DOMAIN_NAME}/channelCapture/chatbotCallback`,
+    },
+  });
+
+  // const updated =
+  await updateChannelWithCapture(channel, {
+    userId,
+    botName,
+    botAlias,
+    controlTaskSid,
+    releaseType,
+    studioFlowSid,
+    releaseFlag,
+    memoryAttribute,
+    chatbotCallbackWebhookSid: chatbotCallbackWebhook.sid,
+  });
+};
+
+type HandleChannelCaptureParams = {
+  channelSid: string; // The channel to capture (in Studio Flow, flow.channel.address)
+  message: string; // The triggering message (in Studio Flow, trigger.message.Body)
+  language: string; // (in Studio Flow, {{trigger.message.ChannelAttributes.pre_engagement_data.language | default: 'en-US'}} )
+  botSuffix: string;
+  triggerType: TriggerTypes;
+  releaseType: ReleaseTypes;
+  studioFlowSid?: string; // The Studio Flow sid. Needed to trigger an API type execution once the channel is released. (in Studio Flow, flow.flow_sid)
+  memoryAttribute?: string; // Where in the channel attributes we want to save the bot's memory (allows usage of multiple bots in same channel)
+  releaseFlag?: string; // The flag we want to set true in the channel attributes when the channel is released
+  additionControlTaskAttributes?: string; // Optional attributes to include in the control task, in the string representation of a JSON
+  controlTaskTTL?: number;
+};
```

Comment:
I think you have this same time defined somewhere else, right?
Can't we local `import` types in this repo? It may work, but I'm not sure.